app/gui/view/component-browser/component-group-view/src/lib.rs

```diff
@@ -0,0 +1,224 @@
+//! An EnsoGL implementation of a Component Group View.
+//!
+//! A Component Group View is displayed as a header (containing the name of a group of Enso
+//! components) and a list of entries below it (names of the components belonging to that group).
+
+#![recursion_limit = "512"]
+// === Standard Linter Configuration ===
+#![deny(non_ascii_idents)]
+#![warn(unsafe_code)]
+// === Non-Standard Linter Configuration ===
+#![warn(missing_copy_implementations)]
+#![warn(missing_debug_implementations)]
+#![warn(missing_docs)]
+#![warn(trivial_casts)]
+#![warn(trivial_numeric_casts)]
+#![warn(unused_import_braces)]
+#![warn(unused_qualifications)]
+
+use ensogl_core::prelude::*;
+
+use enso_frp as frp;
+use ensogl_core::application::Application;
+use ensogl_core::data::color::Rgba;
+use ensogl_core::display;
+use ensogl_core::display::shape::*;
+use ensogl_gui_component::component;
+use ensogl_gui_component::component::ComponentView;
+use ensogl_hardcoded_theme::application::component_browser::component_group as theme;
+use ensogl_list_view as list_view;
+use ensogl_list_view::entry;
+use ensogl_list_view::ListView;
+use ensogl_text as text;
+
+
+
+// ==========================
+// === Shapes Definitions ===
+// ==========================
+
+
+// === Header Background ===
+
+/// The background of the Component Group View's header.
+pub mod header_background {
+    use super::*;
+
+    ensogl_core::define_shape_system! {
+        above = [list_view::selection];
+        (style:Style, color:Vector4) {
+            let sprite_width: Var<Pixels> = "input_size.x".into();
+            let sprite_height: Var<Pixels> = "input_size.y".into();
+            let color = Var::<Rgba>::from(color);
+            let shape = Rect((&sprite_width, &sprite_height)).fill(color);
```

Comment:
if this is just a rect with not rounded corners, why dont you use plane instead? Plane is faster to render

Reply:
For some reason I don't understand, changing this line to use Plane instead:
```rust
            let shape = Plane().fill(color);
```
results in the component rendering with wrong colors - specifically, the color behind the ListView is correct, but the one in the header area is not:

<img width="197" alt="Screenshot 2022-04-13 at 20 47 29" src="https://user-images.githubusercontent.com/273837/163251219-da5f0c24-41fd-420f-ba85-d4db3c017025.png">

Would you have some idea or hint as to what could be the reason for this? I checked what will happen if I use `Plane` in the `shape_system` debug scene, by changing the `shape()` function as below:
```rust
pub fn shape() -> AnyShape {
    let shape = Plane().fill(color::Rgba(0.927, 0.937, 0.913, 1.0));
    shape.into()
}
```
and I also got a rectangle of a different color than expected:
<img width="449" alt="Screenshot 2022-04-13 at 21 04 53" src="https://user-images.githubusercontent.com/273837/163252513-80823655-786d-45e4-accd-f2a4aebd0418.png">

Whereas if I restore the `shape()` function to its original content, then change only the color, as below:
```rust
pub fn shape() -> AnyShape {
    let circle1 = Circle(50.px());
    let circle_bg = circle1.translate_x(-(50.0.px()));
    let circle_sub = circle1.translate_y(-(50.0.px()));
    let rect = Rect((100.0.px(), 100.0.px()));
    let shape = circle_bg + rect - circle_sub;
    let shape = shape.fill(color::Rgba::new(0.927, 0.937, 0.913, 1.0));
    shape.into()
}
```
then I can see the expected color:
<img width="190" alt="Screenshot 2022-04-13 at 21 10 45" src="https://user-images.githubusercontent.com/273837/163253173-85a64b90-afad-42ea-8b01-d853b85f10ba.png">

Reply:
Interesting. Seems like a bug. I will investigate that. But let's use rectangle now + create comment that it will be better to use plane, but there is some colouring bug. May be related to: https://github.com/enso-org/enso/pull/3384